scripts/bert/index.rst

```diff
@@ -23,43 +29,144 @@ BERT for Sentence Classification on GLUE tasks
 GluonNLP provides the following example script to fine-tune sentence classification with pre-trained
 BERT model.
 
-Download the GLUE dataset:
++---------------------+--------------------------------------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------------+
+|                     |                                                  MRPC                                                  |                                                  RTE                                                  |                                                 SST-2                                                 |                                                MNLI-m/mm                                               |
++---------------------+--------------------------------------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------------+
```

Comment:
Could you please help provide references for the models in this table? Thanks 
#1184

Reply:
https://arxiv.org/abs/1810.04805